include/robot_interfaces/sensors/pylon_driver.hpp

```diff
@@ -139,6 +142,25 @@ class PylonDriver : public SensorDriver<CameraObservation>
         }
         return image_frame;
     }
+
+private:
+    void set_camera_configuration(GenApi::INodeMap &nodemap)
+    {
+        Pylon::CFloatParameter exposure_time(nodemap, "ExposureTime");
+        exposure_time.SetValue(1500);
+
+        //Pylon::CBooleanParameter enable_frame_rate(nodemap, "EnableAcquisitionFrameRate");
+        //enable_frame_rate.SetValue(true);
+
+        Pylon::CFloatParameter frame_rate(nodemap, "AcquisitionFrameRate");
+        frame_rate.SetValue(100);
+
+        Pylon::CEnumParameter(nodemap, "BalanceWhiteAuto").SetValue("Once");
```

Comment:
Do we want to set the white balance only once? I just remembered we were setting it adaptively when we were viewing the stream from the PylonViewerApp.

Reply:
As the light conditions in the box should be constant, I think there is no need for permanent adjustment.
Originally I wanted to disable auto-white balance completely but unfortunately the same value leads to slightly different results for the different cameras.